app/javascript/dashboard/components/widgets/conversation/ConversationHeader.vue

```diff
@@ -8,8 +8,8 @@
         :username="currentContact.name"
         :status="currentContact.availability_status"
       />
-      <div class="user--profile__meta">
-        <h3 class="user--name text-truncate">
+      <div class="user--profile__meta" style="max-width: 40rem">
```

Comment:
Please avoid inline styles. I think we can move these to the `<style>` tag at the bottom of the file, which is scoped to load only for this file.

Reply:
Also please make sure the `40rem` will not break on small screen devices.